jmeter/jmeter-java/src/test/java/com/baeldung/jmeter/java/JMeterFromJavaUnitTest.java

```diff
@@ -0,0 +1,103 @@
+package com.baeldung.jmeter.java;
+
+import java.io.File;
+import java.net.URL;
+
+import org.apache.jmeter.control.LoopController;
+import org.apache.jmeter.engine.StandardJMeterEngine;
+import org.apache.jmeter.extractor.JSR223PostProcessor;
+import org.apache.jmeter.protocol.http.sampler.HTTPSampler;
+import org.apache.jmeter.save.SaveService;
+import org.apache.jmeter.testelement.TestPlan;
+import org.apache.jmeter.threads.SetupThreadGroup;
+import org.apache.jmeter.util.JMeterUtils;
+import org.apache.jorphan.collections.HashTree;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+
+class JMeterFromJavaUnitTest {
+
+    private WireMockServer wireMockServer = new WireMockServer(new WireMockConfiguration().port(8080));
```

Comment:
For best practice sake, see if you can move this to a random port, rather than have it on a fixed one.